### PR TITLE
Fix | Copying Pages

### PR DESCRIPTION
**Pull request by davidebukali**
#1223

### CODE REVIEW

**Review by edwinmp — CHANGES_REQUESTED** [review]
Why are the options uppercase to begin with? Couldn't changing them to lowercase fix it? Seems overkill to create a hook simply to fix a wonky coding choice.

**Comment by davidebukali** [comment]
@edwinmp changing to lower case requires a migration which may affect existing pages and also need a data migration to update. This seemed a shorter route. If few pages have used the feature, I think lowercasing would be ideal.

**Comment by edwinmp** [comment]
> @edwinmp changing to lower case requires a migration which may affect existing pages and also need a data migration to update. This seemed a shorter route. If few pages have used the feature, I think lowercasing would be ideal.

This just went live last week, by accident ... I'm sure we can afford a cleaner solution with no disruption. I'm absolutely certain that the other options (not manual) haven't been used. The original ticket is still in review and hasn't been commented on.

**Comment by edwinmp** [comment]
No data migration needed.

File: di_website/publications/mixins.py [hunk 1]
```diff
@@ -14,9 +14,9 @@
 from .utils import WagtailImageField, get_downloads
 
 RELATED_CHOICES = (
-    ('MANUAL', 'Manual'),
-    ('COUNTRY', 'Country'),
-    ('TOPIC', 'Topic')
+    ('Manual', 'Manual'),
+    ('Country', 'Country'),
+    ('Topic', 'Topic')
```
Comment (edwinmp) [inline di_website/publications/mixins.py:19]: This still has upper case letters in it.

**Review by edwinmp — CHANGES_REQUESTED** [review]

**Review by edwinmp — APPROVED** [review]

**Pull request merged**